models.yml

```diff
@@ -8,18 +8,25 @@
 #       At the moment we support only X == 6.
 #     - timestamp: Datetime as a unix timestamp. Why a number? This enables queries
 #       in the DB. And we do not need more precision than 1 second.
-#     - <T>[]: This indicates and arbitrary array of the given type. At the moment
+#     - <T>[]: This indicates an arbitrary array of the given type. At the moment
 #       we support only some types. You can add JSON Schema properties for items
 #       using the extra property `items`
 #     - color: string that must match ^#[0-9a-f]{6}$
 # Relations:
 #     - We have the following types: `relation`, `relation-list`, `generic-relation`
-#       and `generic-relation-list`.
+#       and `generic-relation-list`. All relation-types with a `list`-suffix are **always** realized as sql-statements,
+#       in case of n:m-relations with an intermediary table. 
 #     - Non-generic relations: The simple syntax for such a field
-#       `to: <collection>/<field>`. This is a reference to a collection. The reverse
-#       relation field in this collection is <field>. E. g. in a motion the field `meeting_id`
-#       `reference: <collection(field1, ...)` This can be used, if the foreign key reference
-#        is not the <collection/"id">, where the collection is the same as that from the `to`.
+#       `to: <collection>/<field>`. This is a reference to a collection-field, the "other" side of a relation.
+#           In the relational DB this will be a field in a view filled by an automatically generated sql.
+#           During the development of the relational db one relation can have a `to` and a `reference` property.
+#           In this case the `to` value is only used for historical reason, onky the `reference` property is used
+#           for the schema generation.  
+#           If necessary the sql can be written manually, see `sql`.
+#       `reference: <collection(field1, ...)` This will create a foreign key constraint. Therefore a `reference` is
```

Comment:
- fix the `<>` brackets (and use them either for all variables or for none)
- is it correct that it's possible to give multiple fiel names here? If so, how should that work? In this case, an explanation in the docs would be helpful, although I'm not sure if we even need this feature. If it's not possible or if you agree that we do not need this feature, please correct the description
- Currently, the `id` field is the only field that is ever used here. Since all models have an id and one normally only wants to reference models by their id, I don't see this changing anytime soon. Therefore, we could remove the need to append the `(id)` suffix for all references and simply write `reference: <collection>`. Referencing anything other than the id sounds like a bad design idea, anyway. What do you think?